### ISSUE TITLE
Slashing Load Testing

### ISSUE THREAD

**Issue by eskimor** [paritytech/polkadot#7302]
There are concerns that slashing might become to heavy under heavy dispute load. We should:

- [ ] Check the code, whether those concerns are still justified/talk to relevant people.
- [ ] Do a maximum load test on Versi: Significant number of validators, with maximum number of nominators - run malus nodes with maximum dispute load and see what happens.

**Comment by sandreim** [comment]
Unfortunately, we don't have staking on Versi.

**Comment by eskimor** [comment]
I am aware, but for testing purposes it should be possible to reset the network to a chain spec having staking - no?

**Comment by sandreim** [comment]
Yeah that is doable for sure. We should already be having some infra in place to fund the validator accounts. CC @PierreBesson .

**Comment by PierreBesson** [comment]
@sandreim Yes the testnet-manager already support setting up PoS validators and soon nominators as well https://github.com/paritytech/testnet-manager/pull/84
